components/NumericInput/NumericInput.behavior.comp.cy.js

```diff
@@ -61,6 +61,166 @@ describe('Test the NumericInput component behavior', () => {
+        invalidFeedbackText: 'Test feedback text',
+        value: 0,
+        incDecValues: [1, 10, 100],
+        onReady: readySpy,
+      },
+    });
+
+    cy.get('@readySpy')
+      .should('have.been.calledOnce')
+      .then(() => {
+        cy.get('[data-cy="numeric-decrease-sm"]').should('be.disabled');
+        cy.get('[data-cy="numeric-decrease-md"]').should('be.disabled');
+        cy.get('[data-cy="numeric-decrease-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-increase-sm"]').click();
+
+        cy.get('[data-cy="numeric-decrease-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-decrease-md"]').should('be.disabled');
+        cy.get('[data-cy="numeric-decrease-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-increase-md"]').click();
+
+        cy.get('[data-cy="numeric-decrease-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-decrease-md"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-decrease-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-increase-lg"]').click();
+
+        cy.get('[data-cy="numeric-decrease-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-decrease-md"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-decrease-lg"]').should('not.be.disabled');
+      });
+  });
+
+  it('Test increase buttons are disabled', () => {
+    const readySpy = cy.spy().as('readySpy');
+
+    cy.mount(NumericInput, {
+      props: {
+        label: 'Test',
+        invalidFeedbackText: 'Test feedback text',
+        value: 1000,
+        incDecValues: [1, 10, 100],
+        onReady: readySpy,
+        maxValue: 1000,
+      },
+    });
+
+    cy.get('@readySpy')
+      .should('have.been.calledOnce')
+      .then(() => {
+        cy.get('[data-cy="numeric-increase-sm"]').should('be.disabled');
+        cy.get('[data-cy="numeric-increase-md"]').should('be.disabled');
+        cy.get('[data-cy="numeric-increase-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-decrease-sm"]').click();
+
+        cy.get('[data-cy="numeric-increase-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-increase-md"]').should('be.disabled');
+        cy.get('[data-cy="numeric-increase-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-decrease-md"]').click();
+
+        cy.get('[data-cy="numeric-increase-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-increase-md"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-increase-lg"]').should('be.disabled');
+
+        cy.get('[data-cy="numeric-decrease-lg"]').click();
+
+        cy.get('[data-cy="numeric-increase-sm"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-increase-md"]').should('not.be.disabled');
+        cy.get('[data-cy="numeric-increase-lg"]').should('not.be.disabled');
+      });
+  });
+
+  it('Input of a larger value replaces the initial value', () => {
```

Comment:
This test and the next one have the same `it` message.  They should be edited to be descriptive of what they are specifically testing, without getting too long.

Reply:
Also seems like there should be a similar test for the medium increase button.

Reply:
I've updated both these tests to have a single increment value either larger or smaller than the initial value. This tests all cases as the logic to determine if the initial value should be replaced all happens in the the same function.  It simply depends on the `incDecValues[]` defined by the form.